node-data/src/events.rs

```diff
@@ -0,0 +1,65 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use crate::ledger::{Block, SpentTransaction, Transaction};
+
+#[derive(Clone)]
+pub struct NodeEvent {
```

Comment:
Remember to [derive `Debug` for public items](https://rust-lang.github.io/api-guidelines/debuggability.html#c-debug)